metron-platform/metron-indexing/README.md

```diff
@@ -69,13 +69,17 @@ Depending on how you start the indexing topology, it will have either Elasticsea
 | `batchTimeout`       | The timeout after which a batch will be flushed even if `batchSize` has not been met. | Defaults to a duration which is a fraction of the Storm parameter `topology.message.timeout.secs`, if left undefined or set to 0.  Ignored if batchSize is `1`, since this disables batching.|
 | `enabled`            | A boolean indicating whether the writer is enabled.                                   | Defaults to `true`                                                                                                                                    |
 | `fieldNameConverter` | Defines how field names are transformed before being written to the index.  Only applicable to `elasticsearch`.          | Defaults to `DEDOT`.  Acceptable values are `DEDOT` that replaces all '.' with ':' or `NOOP` that does not change the field names . |
+| `metronId`           | A boolean indicating whether the writer should use the id generated by Metron         | Defaults to `false`.  This setting only applies to Elasticsearch, the id used with Solr is configured in the Solr schemas.
```

Comment:
This setting defines what the Elasticsearch document ID is set to.  I am wondering if we should use a slightly different name for the setting that is more descriptive of that.  We could call it `documentId` or `setDocumentId` that still takes a true/false value like you have.  

To use the Metron GUID...
```
{
  "elasticsearch": {
    "index": "bro",
    "setDocumentId": true
  }
}
```

Reply:
Can you also call out in your docs and/or code that this is about setting the Elasticsearch document ID, instead of just using the generic term ID?  I think that would be more clear.

Reply:
Done.  Did I miss any places where we should use "document ID"?